stdlib/ballerina-grpc/src/main/java/org/ballerinalang/net/grpc/stubs/DefaultStreamObserver.java

```diff
@@ -0,0 +1,274 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ballerinalang.net.grpc.stubs;
+
+import com.google.protobuf.Descriptors;
+import io.grpc.stub.StreamObserver;
+import org.ballerinalang.bre.Context;
+import org.ballerinalang.connector.api.ConnectorUtils;
+import org.ballerinalang.connector.api.Executor;
+import org.ballerinalang.connector.api.ParamDetail;
+import org.ballerinalang.connector.api.Resource;
+import org.ballerinalang.connector.api.Service;
+import org.ballerinalang.model.types.BStructType;
+import org.ballerinalang.model.types.BType;
+import org.ballerinalang.model.values.BBoolean;
+import org.ballerinalang.model.values.BFloat;
+import org.ballerinalang.model.values.BInteger;
+import org.ballerinalang.model.values.BRefType;
+import org.ballerinalang.model.values.BString;
+import org.ballerinalang.model.values.BStruct;
+import org.ballerinalang.model.values.BValue;
+import org.ballerinalang.net.grpc.BallerinaGrpcServerConnector;
+import org.ballerinalang.net.grpc.Message;
+import org.ballerinalang.net.grpc.MessageConstants;
+import org.ballerinalang.net.grpc.MessageRegistry;
+import org.ballerinalang.net.grpc.MessageUtils;
+import org.ballerinalang.net.grpc.exception.GrpcClientException;
+import org.ballerinalang.net.grpc.exception.UnsupportedFieldTypeException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * This is Stream Observer Implementation for gRPC Client Call.
+ */
+public class DefaultStreamObserver implements StreamObserver<Message> {
+    private static final Logger LOGGER = LoggerFactory.getLogger(DefaultStreamObserver.class);
```

Comment:
Please use consistent variable name for these loggers. Please check other cases.